.github/workflows/sonar-analysis.yaml

```diff
@@ -0,0 +1,37 @@
+###
+#
+# Run the SonarCloud Code Analysis on each pull request and on the main branch
+#
+# https://sonarcloud.io/project/configuration/GitHubActions?id=swisspost_common-web-frontend
+#
+###
+
+name: SonarCloud Code Analysis
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Should this (and maybe also other workflows) also run on "ready_for_review"?
See "Webhook payload object for pull_request" on the page: https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=ready_for_review#pull_request